docs/modules/deployment/pages/upgrade/backup/backup-debian-ubuntu.adoc

```diff
@@ -0,0 +1,36 @@
+[[backup-debian-ubuntu]]
+= Backup OpenNMS {page-component-title} in Debian / Ubuntu installations
```

Comment:
```suggestion
= Backup OpenNMS {page-component-title} in Debian/Ubuntu installations
```
If we go for the tab structure for each OS as with the installation section, we can remove the OS from the title.

Reply:
removed it